src/lib/yang/alarms.lua

```diff
@@ -1,5 +1,53 @@
 module(..., package.seeall)
 
+local data = require('lib.yang.data')
+local util = require('lib.yang.util')
+
+local state = {
+   alarm_inventory = {},
+}
+
+function get_state ()
+   return {
+      alarm_inventory = state.alarm_inventory,
+   }
```

Comment:
I wonder here -- you return a fresh object wrapper but then the alarm_inventory itself is the singleton.  If that's ok, why not just return `state`?  If it's not OK, then why not, and you'd need to copy the alarm_inventory.